cogs/dank_memes.py

```diff
@@ -116,18 +116,19 @@ class DankMemes(BaseCog):
         if not ctx.message.attachments:
             await ctx.send('No file detected')
         else:
+            images = sorted(os.listdir(MEDIA_PATH))
             async with ctx.typing():
                 for attachment in ctx.message.attachments:
                     file = await download(attachment.url)
                     await self._process_image(file)
-                    await ctx.send('Dank image uploaded successfully')
+                    await ctx.send(f'Dank image {len(images) + 1} uploaded successfully')
```

Comment:
Maybe rephrase this to something like "dank image uploaded successfully at index {index}"